performance/tests/ListItemsTests.js

```diff
@@ -0,0 +1,135 @@
+/* global page, minFPS, maxFID, maxCLS, stepNumber, testMultiple, maxDCL, maxFCP, maxLCP, passRatio */
+/* eslint-disable*/
+const TestResults = require('../TestResults');
+const {CLS, FID, FPS, getAverageFPS, PageLoadingMetrics} = require('../TraceModel');
+const {clsValue, firstInputValue, getFileName, scrollAtPoint} = require('../utils');
+
+const listItemTests = (componentName, dataSize) => describe(componentName, () => { // eslint-disable-line
+	jest.setTimeout(100000);
```

Comment:
Could you please address why we need this?

Reply:
I had to change the default timeout of 30000 because otherwise the test would fail